src/Homework4/Loor_back.py

```diff
@@ -0,0 +1,21 @@
+"""
+Оглянемся назад
+Даны два натуральных числа.
+Вычислите их наибольший общий делитель
+при помощи алгоритма Евклида (мы не знаем функции и рекурсию).
+"""
+
+
+def evklid(number1, number2):
+
+    while number1 and number2:
+        if number1 > number2:
+            number1 = number1 % number2
+        else:
+            number2 = number2 % number1
+    return number1 + number2
```

Comment:
it's also just a hack )))

Reply:
This is the first solution on the internet)